cypress/objects/logingPage.js

```diff
@@ -0,0 +1,24 @@
+class LoginPage{
+    visit(){
+        cy.visit('/')
+    }
+    getValidationError(){
+        return cy.get('.Select.not-valid')
+    }
+    getSubmit(){
```

Comment:
It would be great to add 'Button' to the end, as in `getSubmitButton()`. This way you'll know for sure what kind of element you're talking about, in this case a button :)